src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -41,6 +41,15 @@ export const PeoplePage: React.FC = () => {
     }
   };
 
+  const fields = [
+    { id: 1, fieldName: 'Name' },
+    { id: 2, fieldName: 'Sex' },
+    { id: 3, fieldName: 'Born' },
+    { id: 4, fieldName: 'Died' },
+    { id: 5, fieldName: 'Mother' },
+    { id: 6, fieldName: 'Father' },
+  ];
+
```

Comment:
Use `useMemo` or move it from the component.

Also, you could try `Object.keys(people[0])` and capitalize words